def/zh-cn/menu_darwin.cson

```diff
@@ -27,197 +27,196 @@ Menu:
       "Show All":
         value: "Show All"
       Quit:
-        value: "Quit"
+        value: "退出"
   File:
-    value: "File"
+    value: "文件"
     submenu:
       "New Window":
-        value: "New Window"
+        value: "打开新的窗口"
       "New File":
-        value: "New File"
+        value: "新建文件"
       "Open…":
-        value: "Open..."
+        value: "打开..."
       "Add Project Folder…":
-        value: "Add Project Folder..."
+        value: "添加项目文件夹..."
       "Reopen Last Item":
-        value: "Reopen Last Item"
+        value: "重新打开关闭的标签页"
       Save:
-        value: "Save"
+        value: "保存"
       "Save As…":
-        value: "Save As..."
+        value: "另存为..."
       "Save All":
-        value: "Save All"
+        value: "全部保存"
       "Close Tab":
-        value: "Close Tab"
+        value: "关闭标签页"
       "Close Pane":
-        value: "Close Pane"
+        value: "关闭面板"
       "Close Window":
-        value: "Close Window"
+        value: "关闭窗口"
       "Close All Tabs":
-        value: "Close All Tabs"
+        value: "关闭所有标签页"
   Edit:
-    value: "Edit"
+    value: "编辑"
     submenu:
       Undo:
-        value: "Undo"
+        value: "撤销"
       Redo:
-        value: "Redo"
+        value: "恢复"
       Cut:
-        value: "Cut"
+        value: "剪切"
       Copy:
-        value: "Copy"
+        value: "复制"
       "Copy Path":
-        value: "Copy Path"
+        value: "复制文件路径"
       Paste:
-        value: "Paste"
+        value: "粘贴"
       "Select All":
-        value: "Select All"
+        value: "全选"
       "Toggle Comments":
-        value: "Toggle Comments"
+        value: "注释/取消注释"
       Lines:
-        value: "Lines"
+        value: "行操作"
         submenu:
-          Indent:
-            value: "Indent"
+            value: "缩进"
           Outdent:
-            value: "Outdent"
+            value: "退格"
           "Auto Indent":
-            value: "Auto Indent"
+            value: "自动缩进"
           "Move Line Up":
-            value: "Move Line Up"
+          value: "上移当前行"
```

Comment:
Recently a [Travis automatic test](https://travis-ci.org/liuderchi/atom-i18n) has been integrated with Pull Request review process.
If there is any syntax error we can know before merging. :+1: